src/amo/components/AddonMoreInfo/index.js

```diff
@@ -180,18 +173,6 @@ export class AddonMoreInfoBase extends React.Component<InternalProps> {
           {i18n.gettext('Read the license agreement for this add-on')}
         </Link>
       ) : null,
-      versionHistoryLink: addonHasVersionHistory(addon) ? (
-        <li>
-          <Link
-            className="AddonMoreInfo-version-history-link"
-            to={addQueryParams(`/addon/${addon.slug}/versions/`, {
-              src: location.query.src,
-            })}
-          >
-            {i18n.gettext('See all versions')}
-          </Link>
-        </li>
```

Comment:
So, we need this link all the time now. I suggest adding it back like this:

```js
      versionHistoryLink: (
        <li>
          <Link
            className="AddonMoreInfo-version-history-link"
            to={addQueryParams(`/addon/${addon.slug}/versions/`, {
              src: location.query.src,
            })}
          >
            {i18n.gettext('See all versions')}
          </Link>
        </li>
       )
```

Reply:
Added this back. Works fine.